internal/pkg/mapper/configmetadata/remote_test.go

```diff
@@ -16,7 +16,6 @@ import (
 
 func TestConfigMetadataMapAfterRemoteLoad(t *testing.T) {
 	t.Parallel()
-	t.Helper()
```

Comment:
To som minule prehliadol, `Helper` sa pouziva iba v nejakych v pomocnych metodach testu, nie priamo v teste, viac info https://pkg.go.dev/testing#B.Helper